sphinx/cmd/quickstart.py

```diff
@@ -260,7 +280,7 @@ def ask_user(d):
                    'selected root path.'))
         print('sphinx-quickstart will not overwrite existing Sphinx projects.')
         print()
-        do_prompt(d, 'path', 'Please enter a new root path (or just Enter '
+        d['path'] = do_prompt('Please enter a new root path (or just Enter '
```

Comment:
Good improvement! Could you fix indentation of hanging lines?

Reply:
Will do